### PR TITLE
Shantanu

### PR DESCRIPTION
**Pull request by shaantanu9**
Please merge my code in main branch

### CODE REVIEW

File: inin.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>Shantanu</p>
```
Comment (Gourav1221) [inline inin.html:15]: add new line it is not perfect yet and do the styling part also

**Review by Gourav1221 — COMMENTED** [review]
do it asap

File: inin.html (hunk 1 above)
Comment (shaantanu9) [reply inin.html:15]: Cimpoleted

**Review by shaantanu9 — COMMENTED** [review]
ok done please review

**Pull request merged**